lib/jekyll_auth/config.rb

```diff
@@ -3,21 +3,36 @@ def self.config_file
     File.join(Dir.pwd, '_config.yml')
   end
 
+  def self.jekyll_config
+    @config ||= begin
+      jekyll_config = YAML.safe_load_file(config_file)
+    rescue
+      {}
+    end
+  end
+
+  def self.destination
+    @config ||= begin
+      JekyllAuth.jekyll_config['destination'] || '_site'
```

Comment:
To keep things consistent, I'd make the fallback `File.expand_path('_site', Dir.pwd)`